CrawlJob.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Simgroep\ConcurrentSpiderBundle;
+
+use PhpAmqpLib\Message\AMQPMessage;
+
+class CrawlJob
+{
+    /**
+     * @var string
+     */
+    private $url;
+
+    /**
+     * @var string
+     */
+    private $baseUrl;
+
+    /**
+     * @var array
+     */
+    private $blacklist = array();
```

Comment:
remove this value set to array because its already done in constructor, so by default is array()